pom.xml

```diff
@@ -17,7 +17,7 @@
     <parent>
         <artifactId>maven-parent-pom</artifactId>
         <groupId>org.eclipse.che.parent</groupId>
-        <version>6.12.1</version>
+        <version>6.13.0</version>
```

Comment:
Why we don't use the last released version 6.14.0

Reply:
I think there is no difference between 6 13 0 and 6 14 0 for this project